Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG IRONLIB_REPO=ghcr.io/metal-toolbox
-ARG IRONLIB_TAG=sha256:c06b454b0264a6837cefaf5ba933d0d50c249aa8cbfec6c2ce2e76decb65a02f
-FROM $IRONLIB_REPO/ironlib@$IRONLIB_TAG as intermediate
+ARG IRONLIB_TAG=v0.2.11
```

Comment:
maybe do v0.2.11@sha256:... so that re-tags won't break us by mistake (mostly just for good hygiene)

Reply:
Just checked and it should be `ghcr.io/metal-toolbox/ironlib:v0.2.11@sha256:8b3c5a78a2b323860b0525723aad7451a7e026a5e552b3ce945522762fe0870d`

![image](https://github.com/metal-toolbox/vogelkop/assets/708570/a8042866-8529-47fb-af8e-0f764d751a92)